src/elements/video/demo/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html ng-app="demo">
+<head lang="en">
+  <meta charset="UTF-8">
+  <title></title>
+  <link rel="stylesheet" href="../../../../node_modules/semantic-ui-css/semantic.css"/>
+  <style>
+    body {
+      padding: 2em;
+    }
+  </style>
+</head>
+<body ng-controller="VideoDemoCtrl">
+
+  <div class="ui stackable centered grid">
+    <div class="row">
+      <div class="six wide column">
+        <h1 class="ui header">Youtube</h1>
+        <sm-video source="youtube" data-id="O6ROCaLvMnQ" placeholder="http://placekitten.com/601/300" pr-theme="light" options="options"
+                  on-ready="onReady(player)" pr-enablejsapi="1"></sm-video>
+      </div>
+    </div>
+    <div class="row">
+      <div class="six wide column">
```

Comment:
@mxth I think that if fine to have play being loaded after placeholder, good point about performance.

It works fine in Vimeo case for me. Controls are loaded and function normally. However, when I play Youtube video I get a bunch of errors:
`GET chrome-extension://eojlgccfgnjlphjnlopmadngcgmmdgpk/cast_sender.js net::ERR_FAILED`